### ISSUE TITLE
Adding a commandID for navigation in the command palette so that it can be changed,

### ISSUE THREAD

**Issue by InsigMath** [sumatrapdfreader/sumatrapdf#4676]
**Is your feature request related to a problem? Please describe.**
When I open up the command palette I have to use the arrow keys to scroll up and down the list of commands (or tabs and history) and I have no way of changing the navigation controls of the palette.

**Describe the solution you'd like**
I would like some command that I can override that allows me to change the control from the arrow keys to something like Tab/Shift+Tab, this would emulate my workflow I would use for code completion used in different IDEs.

**Additional context**
I am using the Pre-release version of the tool (3.6.16779 64-bit) on windows 10.

If this is already possible please let me know as I have not been able to find a corresponding command ID to make the change, thank you!

**Comment by InsigMath** [comment]
I just did a bit more investigated and there is actually a way to change the navigation controls for the palette, its CmdNextTabSmart and CmdPrevTabSmart. I am going to close this ticket!

**Comment by InsigMath** [comment]
I apologize this is actually still not working. When I change the command for CmdNext(Prev)Tab it launches the tab menu but I still require either using the arrow keys or Ctrl+Tab/Ctrl+Shift+Tab whereas I want to be able to just use Tab/Shift+Tab. I looked at the code and it seems that these controls ard hardcoded into the Command Palette, I wonder if it would be trivial to expose a command that allows users to change those navigation controls.

**Comment by GitHubRulesOK** [comment]
hmm most commands can be custom reassigned so currently.

<h2 id="tabs" style="box-sizing: border-box; margin: 1rem 0px 0.6rem; transition: background-color 1s; letter-spacing: -0.01em; font-weight: 600; font-size: 1.5rem; color: rgb(0, 0, 0); font-family: sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Tabs<a class="hlink" href="https://www.sumatrapdfreader.org/docs/Commands#tabs" style="box-sizing: border-box; color: transparent; text-decoration: none; text-underline-offset: 4px;"><span> </span>#</a></h2>
Command IDs | Keyboard shortcuts | Command Palette | Notes
-- | -- | -- | --
CmdNextTabSmart | Ctrl + Tab | Smart tab Switch | ver 3.6+
CmdPrevTabSmart | Ctrl + Shift + Tab | Smart tab Switch | ver 3.6+

<br class="Apple-interchange-newline">

I was going to suggest the following but due to the dialog popping up it does not work in current pre-release
@kjk I think in a previous version the CTRL K listing was not automatically invoked ?
```
Shortcuts [
	[
		Cmd = CmdPrevTabSmart
		Key = Shift + Tab
		ToolbarText = <Tab
	]
	[
		Cmd = CmdNextTabSmart
		Key = Tab
		ToolbarText = Tab>
	]

]
```

**Comment by InsigMath** [comment]
Yeah I haven't really continued exploring this issue and will simply use the controls that are available. If the contributers to sumatra feel there really isn't any real way to provide that command exposure without some major changes then I am alright closing the ticket. In any case thank you @GitHubRulesOK for the response.

**Comment by kjk** [comment]
That's beyond the capability of our system
